anaconda.py

```diff
@@ -201,6 +201,18 @@ def parse_arguments(argv=None, boot_cmdline=None):
     namespace = ap.parse_args(argv, boot_cmdline=boot_cmdline)
     return (namespace, ap.deprecated_bootargs)
 
+def check_arguments(namespace, logger):
+    from pyanaconda.argument_parsing import ArgumentChecker
+    from pyanaconda.errors import AnacondaArgumentError
+
+    ac = ArgumentChecker()
+
+    try:
+        ac.validate_args(namespace)
+    except AnacondaArgumentError as e:
+        print("")  # print blank line to make the message easier to read
+        exit_on_error(logger, str(e))
```

Comment:
Please, consider https://docs.python.org/3/library/argparse.html#exiting-methods to report the errors.

Reply:
The point is that this error is not only thanks to bad arguments. It could be because some resources are not valid or present or similar.